lib/changeful/helpers/view_helpers.rb

```
@@ -26,7 +26,7 @@ module ViewHelpers
     # 
     # @return [String] a content based on a key
     def changeful_content(key, options = {})
-      contents ||= Content.all_contents_in(current_view)
+      current_view.present? ? contents ||= Content.all_contents_in(current_view) : raise 
```

You should raise an exception with a meaning message.